src/libraries/Microsoft.Extensions.Options/tests/SourceGenerationTests/Baselines/NetFX/Validators.g.cs

```diff
@@ -607,7 +682,7 @@ internal sealed partial class __ThirdModelValidator__
 }
 namespace Nested
 {
-    partial record struct Container7 
+    partial record struct Container7
```

Comment:
It looks like the current emitter produces extra whitespace at the end of declarations. I opened this file to view it and ended up saving with formatting which remove this extraneous whitespace.